streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -323,21 +322,6 @@ public void shouldRestoreRecordsUpToHighwatermark() {
         assertEquals(2, stateRestoreCallback.restored.size());
     }
 
-    @Test
-    public void shouldRecoverFromInvalidOffsetExceptionAndRestoreRecords() {
```

Comment:
This test was broken: it throw the `InvalidOffsetException` only once and thus the second `poll()` succeeds -- however, this is not how a real consumer behalves.